make wrong db_constraint setting an error in checks (#94)
Thermondo/django-heroku-connect: docs/checks.rst

Read-write mappings in Heroku Connect need to provide an upsert field to
identify the record.

``heroku_connect.E008``
~~~~~~~~~~~~~~~~~~~~~~~

A related field (:class:`ForeignKey<django.db.models.ForeignKey>` or
:class:`ManyToManyField<django.db.models.ManyToManyField>`) pointing to

> a Heroku Connect model should not use database-constraints.

Reviewer: IMHO you could increase the literal strictness too.
```suggestion
a Heroku Connect model may not use database-constraints.
```

> In some cases like version updates Heroku Connect does recreate the tables,
> which breaks if there are database constraints pointing to them.

Reviewer: Teeny tiny spelling suggestion:
```suggestion
In some cases, such as version updates, Heroku Connect recreates the tables,
which will break if there are database constraints pointing to them.
```

Read-write mappings in Heroku Connect need to provide an upsert field to
identify the record.

``heroku_connect.E008``
~~~~~~~~~~~~~~~~~~~~~~~

A related field (:class:`ForeignKey<django.db.models.ForeignKey>` or
:class:`ManyToManyField<django.db.models.ManyToManyField>`) pointing to
a Heroku Connect model may not use database-constraints.

In some cases, such as version updates, Heroku Connect recreates the tables,

> which will break if there are database constraints pointing to them.

Reviewer 2: I think, it would be good to mention official HC doc about this somewhere...
https://devcenter.heroku.com/articles/heroku-connect-logs-errors#error-during-sync-psycopg2-integrityerror-update-or-delete-on-table-mycustomobject__c-violates-foreign-key-constraint
